### PR TITLE
Dev ag passport

### PR DESCRIPTION
**Pull request by gonzalezad1**

### CODE REVIEW

File: pages/forgot-password.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+);
+
+const ForgotPassword = () => (
+  <div style={{ display: "block", backgroundColor: "#cad4de", minHeight: "100%" }}>
+    <Navigation />
+    <ForgotPasswordContainer style={{ marginTop: 100 }} />
+    <Footer fixed />
+  </div>
+);
+export default ForgotPassword;
```
Comment (revolunet) [inline pages/forgot-password.js]: Sinon il y a le composant `src/common/Layout`

File: src/components/communComponents/Api.js [hunk 2]
```diff
@@ -5,11 +5,40 @@ const API_URL = process.env.API_URL || "http://127.0.0.1:4000";
...
+    return !!token;
+  };
+
   const fetchParams = {
     method: "GET",
     credentials: "include",
     ...params
   };
+
+  if (loggedIn()) {
```
Comment (revolunet) [inline src/components/communComponents/Api.js]: `hasToken()`

File: src/components/loginComponents/LoginForm.js [hunk 3]
```diff
@@ -105,6 +105,7 @@ export const LoginFormView = ({ formData, onSubmit, error, status }) => (
           "Me connecter"}
       </button>
       <br />
+      <a href="forgot-password">J'ai oublié mon mot de passe</a>
```
Comment (revolunet) [inline src/components/loginComponents/LoginForm.js]: `/forgot-password`

File: cypress/integration/login.spec.js [hunk 4]
```diff
@@ -3,36 +3,67 @@ require("./utils");
...
+  });
 
+  it("Forgot password process", function() {
+    cy.visit("/reset-password?token=LpWpzK4Jla9I87Aq");
+    cy.get("#root_newPassword").type("udud");
+    cy.get("#root_verifyPassword").type("ud");
     cy.get("button.btn-success").click();
 
-    cy.get("div.alert-danger").should("contain", "Impossible de se connecter");
+    //cy.location("pathname", { timeout: 10000 }).should("eq", "/login");
```
Comment (douglasduteil) [inline cypress/integration/login.spec.js]: What do you expect at the end of the test ?

File: cypress/integration/mandataires.spec.js [hunk 5]
```diff
@@ -6,23 +6,31 @@ describe("Mandataires", function() {
...
         it("counter should show 2/3", () => {
-          cy.visit("/mandataires");
+          cy.visit("/mandataires", {
+            Authorization: "Bearer " + localStorage.getItem("id_token")
+          });
           cy.get(".react-tabs .react-tabs__tab-list").should("contain", "2 / 3");
         });
         it("can add new mesure", () => {
-          cy.visit("/mandataires");
+          console.log("lo2", localStorage);
```
Comment (douglasduteil) [inline cypress/integration/mandataires.spec.js]: debug to remove

File: cypress/integration/mandataires.spec.js [hunk 6]
```diff
@@ -41,7 +49,7 @@ describe("Mandataires", function() {
           cy.get(".react-tabs .rt-tr-group").should("have.length", 3);
         });
         it("should have new added mesure", () => {
-          cy.visit("/mandataires");
+          cy.visit("/mandataires", { Authorization: "Bearer " + localStorage.getItem("id_token") });
```
Comment (douglasduteil) [inline cypress/integration/mandataires.spec.js]: Did you "lint" this file ? This line is inconsistent comparing to the others

File: cypress/integration/utils.js [hunk 7]
```diff
@@ -11,7 +11,35 @@ Cypress.Commands.add("loginByForm", (username, password) => {
 
   cy.get("button.btn-success").click();
 });
+//     .then(resp => {
+//   window.localStorage.setItem("jwt", resp.body.user.token);
+// });
```
Comment (douglasduteil) [inline cypress/integration/utils.js]: trailing dead code to remove

**Pull request merged**